src/components/Breadcrumb/index.jsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { Link } from "@reach/router";
+import "./style.scss";
+
+function BreadcrumbItem({ url, name, index}) {
+    return (
+        <li key={index}>
+            <Link to={url}> {name} </Link>
```

Comment:
can we make the last item in the list display the default cursor instead of the pointer? the pointer makes it look clickable, and even though you can click it, you're already on the page so it doesn't make sense to encourage people to click it. 

this is the same thing i mentioned on the demo about the header links not appearing clickable when you're on the page.

let me know if you have questions.